pkg/tokenutil/create_secret_key.go

```diff
@@ -0,0 +1,142 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package tokenutil
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/streamnative/pulsarctl/pkg/cmdutils"
+	"github.com/streamnative/pulsarctl/pkg/tokenutil/internal/algorithm"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/pflag"
+)
+
+func createSecretKey(vc *cmdutils.VerbCmd) {
+	var desc cmdutils.LongDescription
+	desc.CommandUsedFor = "This command is used for creating a secret key."
+	desc.CommandPermission = "This command does not need any permission."
+
+	var examples []cmdutils.Example
+	outputToTerminal := cmdutils.Example{
+		Desc:    "Create a secret key and print it to the terminal",
+		Command: "pulsarctl token create-secret-key",
+	}
+
+	outputWithBase64 := cmdutils.Example{
+		Desc:    "Create a secret key and print it with base64 encode to the terminal",
+		Command: "pulsarctl token create-secret-key --base64",
+	}
+
+	outputToFile := cmdutils.Example{
+		Desc:    "Create a secret key and output to a file",
+		Command: "pulsarctl token create-secret-key --output (file path)",
+	}
+	examples = append(examples, outputToTerminal, outputWithBase64, outputToFile)
+	desc.CommandExamples = examples
+
+	o := make([]byte, 32)
+
+	var out []cmdutils.Output
+	toTerminal := cmdutils.Output{
+		Desc: "normal output",
```

Comment:
Why "n" in "normal output" is lower case? But the first character in other output examples are upper case?